source/content/guides/drupal-9-legacy-project-to-pantheon.md

```diff
@@ -0,0 +1,251 @@
+
+- You are able to run `drush` commands in the existing site.
+
+- You are able to checkout your existing site codebase into your local machine.
+
+- The site does not use a nested docroot.
+
+- The site does not use another package and library manager like [Ludwig](https://www.drupal.org/project/ludwig).
+
+- You have a brand new Drupal Pantheon site to host your project.
+
+## Before You Begin
+
+- The site owner should ensure the trusted host setting is up-to-date. Refer to the [Trusted Host Setting](/settings-php#trusted-host-setting) documentation for more information.
+
+- Set the dev environment site mode to git to be able to perform git operations on this environment.
+
+- Clone your site to your local environment following the `git clone` command from the dashboard.
+
+
+<Alert title="Note"  type="info" >
+
+Commit history: The steps in this process migrate a site, so the new site will no longer maintain its existing commit history.
+
+</Alert>
+
+## Prepare the Local Environment
+
+<Partial file="drupal-9/prepare-local-environment-no-clone-no-alias.md" />
+
+2. Get a local copy of both your new Pantheon site and your existing codebase.
+
+1. This doc uses several commands that depend on the locations of both your existing and new site codebases. To simplify this, set the temporary variables `$SOURCE` and `$DESTINATION` in your terminal session to match your folders location.
+
+   ```bash
+   export SOURCE=/absolute/path/to/source/site/codebase
+   export DESTINATION=/absolute/path/to/codebase/cloned/from/pantheon
+   ```
+
+## Copy Existing Configuration
+
+Copy any existing configuration from the source sitem and update the source path as needed to match your configuration folder:
+
+  ```bash{promptUser:user}
+  cp -r $SOURCE/sites/default/files/config/sync/* $DESTINATION/config/
+  # From $DESTINATION:
+  git add config
+  git commit -m "Pull in configuration from source site"
+  ```
+
+If no files are copied through this step, that's acceptable.
+
+## Add Contributed and Custom Code
+
+In your new project structure, replicate your selection of contributed modules and themes, and any custom modules or themes your development team has created.
+
+### Contributed Code
+
+The goal of this process is to have Composer manage all the site's contrib modules, contrib themes, core upgrades, and libraries (referred to as _contributed code_). The only things that should be migrated from the existing site are custom code, custom themes, and custom modules that are specific to the existing site.
+
+
+#### Modules and Themes
+
+
+The steps here ensure that any modules and themes from [drupal.org](https://drupal.org) are in the `composer.json` `require` list.
+
+Once Composer is aware of all the contributed code, you'll be able to run `composer update` from within the directory to have Composer upgrade all the contributed code automatically.
```

Comment:
The Pantheon dashboard will also update Composer dependencies in addition to updating the files from the upstream.